src/Tempest/Framework/Application/AppConfig.php

```diff
@@ -13,6 +14,7 @@ public function __construct(
         public Environment $environment = Environment::LOCAL,
         public bool $enableExceptionHandling = false,
         public bool $discoveryCache = false,
+        public DatabaseDialect $databaseDialect = DatabaseDialect::SQLITE,
```

Comment:
I think it'll be better to move this to the database driver class, as I mentioned in a previous comment